.github/workflows/py-deb-workflow.yml

```diff
@@ -109,13 +109,14 @@ jobs:
 
     - name: Raise & Pack
       run: |
+        pip install bump-my-version==0.29.0
```

Comment:
please update `registry.aws.cloud.mov.ai/qa/ros-buildtools-noetic:v2.0.16` instead of re-installing the pip package